controllers/tekton/pipeline_controller.go

```diff
@@ -111,55 +103,97 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 }
 
 // SetupWithManager sets up the controller with the Manager.
-func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
+func (r *PipelineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&devopsv2alpha1.Pipeline{}).
 		Complete(r)
 }
 
 // deleteExternalResources deletes any external resources associated with the devopsv2alpha1.Pipeline
-func (r *Reconciler) deleteExternalResources(ctx context.Context, pipeline *devopsv2alpha1.Pipeline) error {
+func (r *PipelineReconciler) deleteExternalResources(ctx context.Context, pipeline *devopsv2alpha1.Pipeline) error {
 	klog.Infof("Pipeline [%s] is under deletion.", pipeline.Name)
 
-	// Firstly, we are to find and delete all the related Tekton Tasks
+	// Firstly, we are to find and delete all the related Tekton Tasks.
 	var tknTaskName string
 	for _, taskSpec := range pipeline.Spec.Tasks {
 		tknTaskName = pipeline.Name + "-" + taskSpec.Name
-		if _, err := r.TknClient.TektonV1beta1().
+		if _, err := r.TknClientset.TektonV1beta1().
 			Tasks(pipeline.Namespace).
 			Get(ctx, tknTaskName, metav1.GetOptions{}); err != nil {
-			klog.Infof("Tekton Task [%s] was not found in namespace %s.", tknTaskName, pipeline.Namespace)
-			// Tekton Task resource does not exist.
-			// We do nothing here.
+			klog.Infof("unable to find Tekton Task [%s] in namespace %s", tknTaskName, pipeline.Namespace)
```

Comment:
klog.Infof -> klog.Errorf

Reply:
It's not an error message, you can see that below we return a nil.